### ISSUE TITLE
mapfishapp - JS build broken ?

### ISSUE THREAD

**Issue by pmauduit** [georchestra/georchestra#1779]
On freshly deployed mapfishapp on ppige-dev (https://dev.ppige-npdc.fr/mapfishapp/):

```
Uncaught TypeError: Cannot read property 'prototype' of undefined
    at Object.Ext.apply.extend (ext-base.js:21)
    at mapfishapp.js:formatted:24733
    at mapfishapp.js:formatted:24852
Uncaught TypeError: Cannot set property 'CUSTOM_BBOX' of undefined
    at (index):121
```
surprisingly, it seems working in debug mode.

**Comment by fvanderbiest** [comment]
Right, this is critical and happens also on a locally built mapfishapp on 16.12 and master.

**Comment by fvanderbiest** [comment]
Seems this is due to https://github.com/camptocamp/jstools/pull/13 and files not being minified in the same order, causing an undefined var.

**Comment by fvanderbiest** [comment]
https://github.com/georchestra/georchestra/pull/1775 was applied on 15.12 => to be fixed from 15.12 on.

**Comment by fvanderbiest** [comment]
Fixed from 15.12 on, with 59c3636
Those relying on docker images / DEB packages / WAR files provided by our CI should wait until tomorrow to upgrade in order to get the fix.
